tests/e2e/app-files/config.yml

```diff
@@ -0,0 +1,3 @@
+default:
+  rhino_log_level: !expr Sys.getenv("RHINO_LOG_LEVEL", "TRACE")
+  rhino_log_file: !expr Sys.getenv("RHINO_LOG_FILE", "log.txt")
```

Comment:
I think this is the best way to test log. App should create a log file at trace level.